tools/update_buildinfo.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+USER_MODULES_H=app/include/user_modules.h
+
+COMMIT_ID="$(git rev-parse HEAD)"
+BRANCH="$(git rev-parse --abbrev-ref HEAD | sed -r 's/[\/\\]+/_/g')"
```

Comment:
If we were to use `-E` instead of `-r` it would run on macOS as well.

> Newer versions of FreeBSD and NetBSD sed support both `-E` and `-r`. OpenBSD sed only supports `-E`.

Reply:
Ahh good catch